### DISCUSSION TITLE
Introduction video.

### DISCUSSION THREAD

**Discussion by kun-lex** [EddieHubCommunity/support#6073] [Q&A]
How important is it, for you profolio to have a one minute video introduction.

**Comment by eddiejaoude** [comment]
It is not important but definitely helps you standout from the crowd 👍

**Reply by Ytiny** [reply]
![image](https://user-images.githubusercontent.com/114656605/213848720-@da7aef1e-b56c-465b-a171-4c7bc3856392.png)

**Comment by Sayyidalijufri** [comment]
# **I'am think It not important**

**Reply by Ytiny** [reply]
- [ ] #![image](https://user-images.githubusercontent.com/114656605/213832254-3bdbb904-94df-40bb-b410-46f33bb2955c.gif)
